x-pack/solutions/observability/plugins/apm/test/scout/ui/fixtures/page_objects/agent_explorer.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { KibanaUrl, ScoutPage } from '@kbn/scout-oblt';
+
+export class AgentExplorerPage {
+  constructor(private readonly page: ScoutPage, private readonly kbnUrl: KibanaUrl) {}
+
+  async goto() {
+    await this.page.goto(
+      `${this.kbnUrl.app(
+        'apm'
+      )}/settings/agent-explorer?kuery=&agentLanguage=&serviceName=&comparisonEnabled=true&environment=ENVIRONMENT_ALL`
+    );
+    await this.page.waitForLoadingIndicatorHidden();
+    await this.page
+      .getByRole('heading', { name: 'Settings', level: 1 })
+      .waitFor({ timeout: 10000 });
+
+    return this.page;
+  }
+
+  async clickAgentToggle() {
+    await this.page.testSubj.locator('apmAgentExplorerListToggle').click();
+  }
+
+  async clickInstance(instanceName: string) {
```

Comment:
This isn't used?

Reply:
Thanks for spotting that. I missed converting it to component test after I removed the test from here - I couldn't test it as it shows data for the last 24 hours and our data is created with a back that and the playright clock api didn't work when I tried ( I see the current date as Date.now() ) and in the test I see `No Agents found` always because of that 
This is the expected from an  oblt cluster for example: 
<img width="961" height="414" alt="image" src="https://github.com/user-attachments/assets/acea7b01-158d-4d8e-a78a-27fe8d7989a8" />